shared/constants/swaps.js

```diff
@@ -64,13 +78,16 @@ const SWAPS_TESTNET_HOST = 'https://metaswap-api.airswap-dev.codefi.network';
 
 const BSC_DEFAULT_BLOCK_EXPLORER_URL = 'https://bscscan.com/';
 const MAINNET_DEFAULT_BLOCK_EXPLORER_URL = 'https://etherscan.io/';
+const POLYGON_DEFAULT_BLOCK_EXPLORER_URL = 'https://api.polygonscan.com/api/';
```

Comment:
We probably want to change this value to `https://polygonscan.com` rather than the API endpoint